lib/util/Map.v3

```diff
@@ -110,6 +110,36 @@ class HashMap<K, V> extends PartialMap<K, V> {
 			}
 		}
 	}
+	// Number of key/value pairs in HashMap (expensive operation - cache the value).
```

Comment:
I think I'd rather keep this out of the main `HashMap` implementation for now; can you do this with an `apply()` in your code for now?